src/language-switcher/language-switcher.php

```diff
@@ -2,7 +2,7 @@
 
 namespace WPML\PB\Elementor\LanguageSwitcher;
 
-class LanguageSwitcher implements \IWPML_Action {
+class LanguageSwitcher implements \IWPML_Backend_Action, \IWPML_Frontend_Action {
```

Comment:
@brucepearson this is not loading the class, I'm getting a bunch of errors. Would you like to check? debug.log https://pastebin.com/myNeusth

Reply:
@amElnagdy 
This is failing - class_implements(): Class \WPML\PB\Elementor\LanguageSwitcher\LanguageSwitcher
Did you run composer install to update autoloading?

Reply:
Not sure if I did. Currently, it's working fine. Thanks @brucepearson